libs/community/langchain_community/utilities/pebblo.py

```diff
@@ -150,6 +150,7 @@ class Doc(BaseModel):
     loader_details: dict
     loading_end: bool
     source_owner: str
+    classifier_location: str
```

Comment:
* Could you update the doc-string?
* The doc-strings should be below the attribute not in the model doc-string

Reply:
Updated all the models with suggested doc-string style.